Documentation/UserTsconfig/Setup.rst

```diff
@@ -138,6 +139,18 @@ neverHideAtCopy
     If set, then the hideAtCopy feature for records in TCE will not be used.
 
 
+.. index:: File list; Show hidden files and folders
+
+showHiddenFilesAndFolders
```

Comment:
Can you give a reference where this comes from?

Reply:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/Authentication/BackendUserAuthentication.php#L1440